tamr_unify_client/client.py

```diff
@@ -145,6 +145,22 @@ def datasets(self):
         """
         return self._datasets
 
+    def create_project(unify, project_config):
+        """
+        Create a Project in Unify
+
+        :param: Project configuration should be formatted as specified in the `Public Docs for Creating a Project <https://docs.tamr.com/reference#create-a-project>`_.
```

Comment:
You'll want to specify the name of the param:
```suggestion
        :param project_config: Project configuration should be formatted as specified in the `Public Docs for Creating a Project <https://docs.tamr.com/reference#create-a-project>`_.
```